pkg/azuredisk/controllerserver.go

```diff
@@ -924,6 +924,7 @@ func (d *Driver) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshotRequ
 	location := d.cloud.Location
 
 	parameters := req.GetParameters()
+	tagValueDelimiter := ","
```

Comment:
just define a var tagValueDelimiter, and then if it's empty, then func `ConvertTagsToMap` would treat as `,`

Reply:
done